src/Jackalope/ObjectManager.php

```diff
@@ -970,6 +970,18 @@ public function checkout($absPath)
         $this->transport->checkoutItem($absPath); //FIXME: what about pending move operations?
     }
 
+
+    public function addLabel($path, $label, $move)
+    {
+        $this->transport->addLabel($path, $label, $move);
+    }
+
+
+    public function removeLabel($path, $label)
```

Comment:
these methods are missing from the VersioningInterface for the transport, right?

adding the methods is a BC break, but only jackrabbit so far implements versions. lets update the branch alias in composer.json to 1.3 and add the methods